tests/services/orders/order_status_service/conftest.py

```diff
@@ -1,16 +1,21 @@
+from datetime import datetime
+from mock import Mock
 import pytest
 from cg.apps.tb.api import TrailblazerAPI
 from cg.apps.tb.dto.summary_response import AnalysisSummary
 
-from cg.services.orders.order_status_service.order_status_service import OrderStatusService
-from cg.store.models import Case, Customer, Order
+from cg.services.orders.order_status_service.order_summary_service import OrderSummaryService
+from cg.store.models import Case, Customer, Order, Sample
 from cg.store.store import Store
 from tests.store_helpers import StoreHelpers
 
 
 @pytest.fixture
-def order_status_service(trailblazer_api: TrailblazerAPI, store: Store):
-    return OrderStatusService(analysis_client=trailblazer_api, store=store)
+def summary_service(trailblazer_api: TrailblazerAPI, store: Store, order: Order):
+    service = OrderSummaryService(analysis_client=trailblazer_api, store=store)
+    service.analysis_client = Mock()
```

Comment:
Why do we make this fixture dependent on the trailblazer_api if we override it with a Mock() immediately?

Reply:
Removed in https://github.com/Clinical-Genomics/cg/pull/3050/commits/769d74ad76ab380540c30de681146519fc29b29b